### PR TITLE
Fix compatibility with PHP 5.3

### PR DESCRIPTION
**Pull request by phoenixrvd**

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -16,4 +16,11 @@
             <directory>./tests/</directory>
         </testsuite>
     </testsuites>
+
+    <filter>
+        <whitelist>
+            <directory>./src/</directory>
+        </whitelist>
+    </filter>
+
```
Comment (mikehaertl) [inline phpunit.xml.dist]: @phoenixrvd What does this have to do with PHP 5.3? If you think, it's useful, could you explain what you need this for? As I see it, this is for code coverage reports - but we don't test those anyway.

Comment (phoenixrvd) [reply phpunit.xml.dist]: Sorry, this has nothing with php5.3. The pull request has now only php 5.3 fixes.

**Comment by mikehaertl** [comment]
Thanks! Released 0.4.5 with this change.
